avionics/common/include/states/landed.h

```diff
@@ -30,6 +30,11 @@ class Landed : public IState {
         constexpr static unsigned int LANDED_POLLING_INTERVAL = 5000;
         Scheduler::get_task(static_cast<int>(TaskID::ReadEvalLog)).period =
             Hal::ms(LANDED_POLLING_INTERVAL);
+
+        // REVIEW(akoen) disabledRepeat() is more performant, but may cause
+        // errors if the camera is turned on immediately before turning off
+        // again. Test this.
```

Comment:
`unschedule()` is probably acceptable here; at this point the rocket has landed and there's nothing else for it to do. It's also not that more performant - there's like 4 tasks it has to search through.